### ISSUE TITLE
GIDS: Audit and update model validators

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#4045]
@dneel-bah commented on [Fri Oct 25 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19837)

As a developer, I need to audit and update the model validators so that the codebase is accurately validating data that is loaded into the system.

## Assumptions:
1. Validators are on models; not all models have validators.

## Acceptance Criteria:
1.  All models are documented with the status of the associated validators in the attached supporting artifact template.
2. If a validator is changing the data without notifying the user (ex: excluding duplicates), the validator is updated to notify the user.
3. In GIDS User is notified of validations for a CSV

## PR
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/516
## Supporting Artifacts:
1. [ValidatorTracker.xlsx](https://github.com/department-of-veterans-affairs/vets.gov-team/files/3772863/ValidatorTracker.xlsx)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/516

**Comment by delli-gatti-michael** [comment]
Local testing complete

**Comment by zurbergram** [comment]
Changes have been merged into staging

**Comment by delli-gatti-michael** [comment]
Blocked by bug #4818 which is covered in the following PR:

https://github.com/department-of-veterans-affairs/gibct-data-service/pull/542

**Comment by delli-gatti-michael** [comment]
Blocking defect has been resolved and all changes are in Staging. Closing this issue
